index.bs

```diff
@@ -132,6 +127,22 @@ spec: HTML53; urlPrefix: https://w3c.github.io/html/
             text: document.domain; url:dom-document-domain
             text: scheme; url: origin-scheme
             text: port; url: origin-port
+-->
+
+spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
+    type: dfn
+        urlPrefix: dom.html
+            for: Document; url: concept-document-feature-policy; text: feature policy
+        urlPrefix: browsers.html
+            text: browsing context; url: browsing-context
+            text: origin; url: concept-origin
+            text: opaque origin; url: concept-origin-opaque
+            text: tuple origin; url: concept-origin-tuple
+            text: document.domain; url:dom-document-domain
+            text: scheme; url: origin-scheme
+            text: port; url: origin-port
```

Comment:
Some links are broken, looks like they should be:

- `origin`: `urlPrefix: origin.html`
- `opaque origin`: `urlPrefix: origin.html`
- `tuple origin`: `urlPrefix: origin.html`
- `scheme`: https://url.spec.whatwg.org/#concept-url-scheme
- `port`: https://url.spec.whatwg.org/#concept-url-port

Reply:
are you building locally? if so, have you done ` bikeshed update` recently?  It builds without errors for me on https://api.csswg.org/bikeshed/

Reply:
Yeah, it builds without errors, but the links don't lead where they're supposed to. For example, [opaque origin](https://html.spec.whatwg.org/multipage/browsers.html#concept-origin-opaque) should be https://html.spec.whatwg.org/multipage/origin.html#concept-origin-opaque

Reply:
doh.  ok, thanks for checking the links. all worked (except scheme and port) for me, apparently due to some redirect magic on the part of the whatwg html spec, but probably better to have 'em not relying on that...  so I've updated those dfn links. Does it look OK now?

Reply:
Oh, I see, there's some JavaScript redirect that was blocked by my NoScript. :smile:

Yes, it's working now!